lib/rules/require-aligned-object-values.js

```diff
@@ -12,10 +40,13 @@ module.exports.prototype = {
             'skipWithFunction': 'ignoreFunction',
             'skipWithLineBreak': 'ignoreLineBreak'
         };
+
```

Comment:
@zxqfox I totally agree with separating style commits from code changes. I did not realize that is what he was talking about, or that I had even done it. That is why I asked for clarification.